exchanges/bybit/ratelimit.go

```diff
@@ -76,281 +68,67 @@ const (
-		GetAssetUniversalTransferListRate:         request.NewRateLimit(time.Second, 2),
-		GetAssetAccountCoinBalanceRate:            request.NewRateLimit(time.Second, 2),
-		GetAssetDepositRecordsRate:                request.NewRateLimit(time.Minute, 300),
-		GetAssetDepositSubMemberRecordsRate:       request.NewRateLimit(time.Minute, 300),
-		GetAssetDepositSubMemberAddressRate:       request.NewRateLimit(time.Minute, 300),
-		GetWithdrawRecordsRate:                    request.NewRateLimit(time.Minute, 300),
-		GetAssetCoinInfoRate:                      request.NewRateLimit(time.Minute, 300),
-		GetExchangeOrderRecordRate:                request.NewRateLimit(time.Minute, 300),
-		InterTransferRate:                         request.NewRateLimit(time.Minute, 20),
-		SaveTransferSubMemberRate:                 request.NewRateLimit(time.Minute, 20),
-		UniversalTransferRate:                     request.NewRateLimit(time.Second, 5),
-		CreateWithdrawalRate:                      request.NewRateLimit(time.Second, 1),
-		CancelWithdrawalRate:                      request.NewRateLimit(time.Minute, 60),
-		UserCreateSubMemberRate:                   request.NewRateLimit(time.Second, 5),
-		UserCreateSubAPIKeyRate:                   request.NewRateLimit(time.Second, 5),
-		UserFrozenSubMemberRate:                   request.NewRateLimit(time.Second, 5),
-		UserUpdateAPIRate:                         request.NewRateLimit(time.Second, 5),
-		UserUpdateSubAPIRate:                      request.NewRateLimit(time.Second, 5),
-		UserDeleteAPIRate:                         request.NewRateLimit(time.Second, 5),
-		UserDeleteSubAPIRate:                      request.NewRateLimit(time.Second, 5),
-		UserQuerySubMembersRate:                   request.NewRateLimit(time.Second, 10),
-		UserQueryAPIRate:                          request.NewRateLimit(time.Second, 10),
-		GetSpotLeverageTokenOrderRecordsRate:      request.NewRateLimit(time.Second, 50),
-		SpotLeverageTokenPurchaseRate:             request.NewRateLimit(time.Second, 20),
-		SpotLeverTokenRedeemRate:                  request.NewRateLimit(time.Second, 20),
-		GetSpotCrossMarginTradeLoanInfoRate:       request.NewRateLimit(time.Second, 50),
-		GetSpotCrossMarginTradeAccountRate:        request.NewRateLimit(time.Second, 50),
-		GetSpotCrossMarginTradeOrdersRate:         request.NewRateLimit(time.Second, 50),
-		GetSpotCrossMarginTradeRepayHistoryRate:   request.NewRateLimit(time.Second, 50),
-		SpotCrossMarginTradeLoanRate:              request.NewRateLimit(time.Second, 20),
-		SpotCrossMarginTradeRepayRate:             request.NewRateLimit(time.Second, 20),
-		SpotCrossMarginTradeSwitchRate:            request.NewRateLimit(time.Second, 20),
+func SetRateLimit() request.RateLimitDefinitions {
+	return request.RateLimitDefinitions{
+		defaultEPL:                               request.NewRateLimitWithToken(time.Second*5 /*See: https://bybit-exchange.github.io/docs/v5/rate-limit*/, 120, 1),
```

Comment:
Looks like this has been upped to 600 according to your link. But double check me please